jdaviz/configs/cubeviz/plugins/slice/slice.py

```diff
@@ -223,6 +223,14 @@ def _on_add_data(self, msg):
             # is updated (if enabled)
             self._on_value_updated({'new': self.value})
 
+    def _viewer_slices_changed(self, value):
+        # the slices attribute on the viewer state was changed,
+        # so we'll update the slider to match which will trigger
+        # the slider observer (_on_slider_updated) and sync across
+        # any other applicable viewers
+        if len(value) == 3:
+            self.slice = float(value[-1])
+
```

Comment:
I don't think this is ever called (and would need to be updated for the generalization of the slice plugin if we do need this code)

Reply:
Slipped through while rebasing, thanks!